ec2-spot-elastic-inference/workshop1-running-amazon-ec2-workloads-at-scale.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash -x
+
```

Comment:
All the same comments as above, This looks like temporary files that are used to automate something, but half of them make no sense (exit codes in between, no comments, no definition of what it does and conditionals that will never execute)